lib/scope.js

```diff
@@ -2781,7 +2781,7 @@ module.exports = {
       await scope.afterStep(scope);
     },  // Ends scope.steps.sign()
 
-    download: async ( scope, filename ) => {
+    download: async ( scope, filename_matcher ) => {
```

Comment:
`filename_matcher` to me implies that it's some more powerful lambda function or regex or something, and not just a partial string. Maybe `partial_filename` would be better.